src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
+/* eslint-disable promise/param-names */
+/* eslint-disable no-console */
 'use strict';
 
-// write your code here
+const div = document.createElement('div');
+
+document.body.append(div);
+
+const divFormat = (element, className, message) => {
```

Comment:
function is an action, so it's better to use verbs
and to make function universal we can add element instead of hard-coding div in its name

```suggestion
const formatElement = (element, className, message) => {
```